src/parallel/par.rs

```diff
@@ -21,11 +21,23 @@ use crate::split_at::SplitPreference;
 #[derive(Copy, Clone, Debug)]
 pub struct Parallel<I> {
     iter: I,
+    min_len: usize,
 }
 
+impl<I> Parallel<I> {
+    pub fn with_min_len(self, min_len: usize) -> Self {
+        Self {
+            min_len,
+            ..self
+        }
+    }
+}
+
+const DEFAULT_MIN_LEN: usize = 1;
```

Comment:
It might be useful to make this default larger than one similar to the existing [`COLLECT_MAX_SPLITS`](https://github.com/rust-ndarray/ndarray/blob/87506b8bec7cc9a265289beaa56d0b1b58ffad4f/src/parallel/impl_par_methods.rs#L50).